RefsViews/Branches/BranchesModel.cpp

```diff
@@ -199,21 +196,14 @@ void BranchesModel::rereadBranches()
     mHeaderTags     = new RefScope( mRoot, tr( "Tags" ) );
 
     RM::Repo* repo = mData->repository();
-
-    // TODO: migrate to RM::Repo
-    Git::Repository gitRepo = repo ? repo->gitRepo() : Git::Repository();
-
-    if( gitRepo.isValid() )
+    if( repo )
     {
-        Git::Result r;
-        Git::ReferenceList sl = gitRepo.allReferences( r );
-        if( !sl.isEmpty() )
-        {
-            for( int i = 0; i < sl.count(); ++i )
-            {
-                const Git::Reference &currentRef = sl[ i ];
-                insertRef( false, currentRef );
-            }
+        foreach ( RM::Branch* info, repo->branches()->childObjects<RM::Branch>() ) {
```

Comment:
Well, I can't see any code where you're actually asking RepoMan for the "Scope" objects :)